app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team_list: list[Player]) -> int:
+    return sum([player.get_rating() for player in team_list])
```

Comment:
Consider using a generator expression instead of a list comprehension to avoid creating a redundant list. Change `[player.get_rating() for player in team_list]` to `(player.get_rating() for player in team_list)` to improve efficiency as per the checklist guidelines .